index.js

```diff
@@ -245,10 +245,12 @@ var output_types_datafields = [ //data fields for the outgoing actions
 		]
 	},
 	{ outputTypeId: 'ffe2b0b6', fields: [ //Outgoing Webhook
-			{ fieldName: 'ip', fieldLabel: 'IP Address', fieldType: 'text' },
+			{ fieldName: 'protocol', fieldLabel: 'Protocol', fieldType: 'dropdown', options: [ { id: 'http://', label: 'HTTP' }, { id: 'https://', label: 'HTTPS'} ] },
+			{ fieldName: 'ip', fieldLabel: 'Ip/URL', fieldType: 'text' },
 			{ fieldName: 'port', fieldLabel: 'Port', fieldType: 'port' },
 			{ fieldName: 'path', fieldLabel: 'Path', fieldType: 'text' },
 			{ fieldName: 'method', fieldLabel: 'Method', fieldType: 'dropdown', options: [ { id: 'GET', label: 'GET' }, { id: 'POST', label: 'POST'} ] },
+			{ fieldName: 'content', fieldLabel: 'Content', fieldType: 'dropdown', options: [ { id: 'JSON', label: 'JSON' }, { id: 'RAW', label: 'RAW'} ] },
```

Comment:
Instead of these names (raw, json) can we use the official content types?

`application/json
`application/x-www-form-urlencoded`
`application/xml`
`text/plain`

And name the field “Content-Type” so it’s clear for all?

Reply:
I can make this modification. what would we want the default to be? Would we want to have an option where the content type is not set?

Reply:
I’m sure the axios library has a default, whether we set it or not. I would say let the default be `application/json` for now.

Reply:
I have added a Default option in the list which does not set the parameter. It will just set it to the axios default of url form encoded.